modules/sov-ibc-transfer/src/rpc.rs

```diff
@@ -22,39 +18,50 @@ impl<S> IbcTransfer<S>
 where
     S: Spec,
 {
-    #[rpc_method(name = "escrowedToken")]
+    #[rpc_method(name = "mintedTokenName")]
```

Comment:
The method name doesn't match.

```diff
- pub fn escrowed_token(
+ pub fn minted_token_name(
```

Reply:
[e9651e1](https://github.com/informalsystems/sovereign-ibc/pull/133/commits/e9651e12c50544acd44a1d4ce4455c65911aebba)